packages/examples/basic/package.json

```diff
@@ -6,18 +6,17 @@
     "build:blade": "cd ../../blade && yarn build",
     "build:app": "DISABLE_ESLINT_PLUGIN=true react-scripts build",
     "analyze:only": "source-map-explorer 'build/static/js/*.js' --gzip",
-    "analyze": "yarn build && yarn analyze:only",
-    "postinstall": "rm -rf node_modules/@razorpay/blade/node_modules"
+    "analyze": "yarn build && yarn analyze:only"
   },
   "dependencies": {
-    "@razorpay/blade": "../../blade",
+    "@razorpay/blade": "../../blade/build",
```

Comment:
Looks like changes from this PR are reverted https://github.com/razorpay/blade/pull/1853. Intentional?

Reply:
did you also add that postinstall script?